docs/guides/deploy-warp-route.mdx

```diff
@@ -72,6 +72,29 @@ For a minimal Warp deployment config example using local anvil chains, see [`war
 
 ## 2. Deployment
 
+### Dry Run a Warp Deployment
+
+Prior to executing a warp deployment, you can execute a dry-run via `--dry-run <CHAIN_NAME>`  or `-d <CHAIN_NAME>` to ensure that the warp deployment will be successful as well as provide analytics on the gas costs of deployment. This will not execute any transactions but will simulate a deployment to display a list of required contracts and their respective gas costs.
```

Comment:
nit: we say the word execute three times back to back, maybe "you can `complete` a dry-run"